examples/petsc/area.c

```diff
@@ -116,7 +116,7 @@ int main(int argc, char **argv) {
     CHKERRQ(ierr);
   } else {
     // Create the mesh as a 0-refined sphere. This will create a cubic surface, not a box
-    ierr = DMPlexCreateSphereMesh(PETSC_COMM_WORLD, topodim, simplex, &dm);
+    ierr = DMPlexCreateSphereMesh(PETSC_COMM_WORLD, topodim, simplex, 1., &dm);
```

Comment:
Question - do we want to update this with a macro, like we did for DMPlexGetClosureIndices?

Reply:
Added them. Thanks